pkg/webui/console/views/application/index.js

```diff
@@ -134,11 +135,10 @@ export default class Application extends React.Component {
   }
 
   render () {
-    const { fetching, error, match, application } = this.props
+    const { fetching, match, error, application } = this.props
 
-    // show any application fetching error, e.g. not found, not rights, etc
     if (error) {
-      return 'ERROR'
+      return <FullViewError error={error} />
```

Comment:
According to this comment https://github.com/TheThingsNetwork/lorawan-stack/issues/659#issuecomment-490437870 we want to avoid this, right?
namely:
> We would then have to use portals to be able to show full page errors. Otherwise, we could only display within the boundaries of the dom that the component is rendered into. E.g. in case of 404 of an entity we would still have to show the sidebar, which is not what we want. And even with portals, it would still render unnecessarily, unless we add logic to prevent that.

here is how the page looks when using this approach:
<img width="1513" alt="Screenshot 2019-05-10 at 17 47 42" src="https://user-images.githubusercontent.com/16374166/57539883-bea90480-734b-11e9-8180-4cbfe3a226e4.png">